test/functional/testlib.bash

```diff
@@ -1979,8 +1983,6 @@ create_config_file() { # swupd_function
 
 	# these values have to be exported in this function because the
 	# TEST_NAME changes during the setup
```

Comment:
I'm confused about moving the setup of the config dir to the top of the file, it was being called from `create_config_file()` because the name of the tests change for each setup, there is even a comment about this here:
```
# these values have to be exported in this function because the
# TEST_NAME changes during the setup
```

Reply:
I'm moving everything that is relative to TEST_NAME to set_env_variables to enforce that everything will be relative to the test. Any file that is generated and needed by the test should be in it's own directory.